test/Microsoft.TestPlatform.PerformanceTests/PerfInstrumentation/PerfAnalyzer.cs

```diff
@@ -2,19 +2,18 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System.Collections.Generic;
+using System.IO;
 
-#if NETFRAMEWORK
 using System;
 using System.Linq;
 
 using Microsoft.Diagnostics.Tracing;
 using Microsoft.Diagnostics.Tracing.Parsers;
 using Microsoft.Diagnostics.Tracing.Session;
-#endif
 
 #nullable disable
```

Comment:
[minor] As you do quite some changes to the class, might be worth to try to remove the `#nullable disable`.

Reply:
There is only so much you can do in 1 day. ;)